package.json

```diff
@@ -16,6 +16,7 @@
   "license": "MIT",
   "scripts": {
     "build": "microbundle --no-sourcemap -f modern,es,cjs",
+    "postbuild": "echo \";Object.defineProperty(exports,'__esModule',{value:true})\" >> dist/index.js",
```

Comment:
@shuding seems to work (checking the SSR sandbox demo), but kinda a dirty workaround..